tensilelite/Tensile/Components/LSU.py

```diff
@@ -245,6 +242,16 @@ def writeReadReduction(self, writer, kernel):
                 DSLoadBX     = DSLoadB32
                 numInstPerVW = bytesPerVector // 4
                 regsPerStore = 1
+            maxOffset =  kernel["LocalSplitU"] * ldsStride + ((numVgprPerLSU // self.LSUfullVw) * numInstPerVW + numInstPerVW) * regsPerStore * (bpr * kernel["WavefrontSize"])
+            if  writer.states.archCaps["HasLDSGT64K"]:
+                if maxOffset >= 65536*2:
```

Comment:
I suggest to simplify the code a bit more.
We do not need if statements for offset.

numAddr = maxOffset // 65536 + 1
addr = writer.vgprPool.checkOut(numAddr ,"addr")